integration/ztoc_test.go

```diff
@@ -17,14 +17,33 @@
 package integration
 
 import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"path/filepath"
 	"strconv"
 	"strings"
 	"testing"
 
 	"github.com/awslabs/soci-snapshotter/soci"
+	"github.com/awslabs/soci-snapshotter/util/dockershell"
+	"github.com/awslabs/soci-snapshotter/util/testutil"
+	"github.com/awslabs/soci-snapshotter/ztoc"
 	v1 "github.com/opencontainers/image-spec/specs-go/v1"
 )
 
+type Info struct {
+	Version   string     `json:"version"`
+	BuildTool string     `json:"build_tool"`
+	Files     []FileInfo `json:"files"`
+}
+
+type FileInfo struct {
+	Filename string `json:"filename"`
+	Offset   int64  `json:"offset"`
+	Size     int64  `json:"size"`
+}
+
```

Comment:
Can we not use them from ztoc/info.go?

Reply:
We can but that would introduce an extra dependency to the root go.mod.

Reply:
If they're in `cmd` pkg, then we can't import it in integration test. Otherwise we'll have circular dependency. (`./cmd` depends on `./`)

Reply:
Yeah, I'm not sure where we should put extra metadata like this